packages/iov-lisk/src/liskconnection.ts

```diff
@@ -93,7 +93,7 @@ export class LiskConnection implements BcpConnection {
     return responseBody.data.height;
   }
 
-  public async postTx(bytes: PostableBytes): Promise<BcpTransactionResponse> {
+  public async postTx(bytes: PostableBytes): Promise<PostTxResponse> {
```

Comment:
postTx -> postTxAndWaitInBlockConfirmation (sames as in bnsconnection postTx)

Reply:
This does not wait for a block. But you can track a posted transaction and observe its confirmations if you want like in this example: https://github.com/iov-one/iov-core/blob/master/packages/iov-lisk/src/liskconnection.spec.ts#L295